src/behaviors/hv-alert/index.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import * as Namespaces from 'hyperview/src/services/namespaces';
+import type { Element, HvComponentOnUpdate } from 'hyperview/src/types';
+import { getBehaviorElements, later } from 'hyperview/src/services';
+import { Alert } from 'react-native';
+
+export default {
+  action: 'alert',
+  callback: (element: Element, onUpdate: HvComponentOnUpdate) => {
+    const title = element.getAttributeNS(Namespaces.HYPERVIEW_ALERT, 'title');
```

Comment:
Should we additionally add type-checks here? There's nothing stopping someone from inserting a boolean or numbers here by accident, and if we explicitly catch that here the debugging will be easier.

Reply:
Looks like getAttributeNS always returns `string` or `null`. In that case, all we have to do is ensure required fields aren't `null`. Out of scope for this PR though

Reply:
There are no flow warnings because these fields are optional in [Alert.alert of RN 0.52.2](https://github.com/facebook/react-native/blob/4080141a93127ba64e2d418c73e95b36f4eb5eee/Libraries/Alert/Alert.js#L44-L45)